app/helpers/application_helper.rb

```diff
@@ -31,4 +31,8 @@ def firm_map_component(center:, adviserPinUrl:, officePinUrl:)
       yield
     end
   end
+
+  def feature_enabled?(flag_name)
+    ENV[flag_name] == 'true'
+  end
```

Comment:
We'll add it to dev. Good shout.

I deliberately left it as opt-in rather than opt-out so we could avoid any issues with the production env not picking up the var which we experienced on CIA.